plugins/org.polarsys.capella.transition.system2subsystem/src/org/polarsys/capella/transition/system2subsystem/activities/FinalizeSubsystemTransitionActivity.java

```diff
@@ -75,6 +79,20 @@ private void unloadTemporaryResource(ActivityParameters activityParams_p) {
       Resource transformationResource = transformationRoot.eResource();
       Resource targetResource = (Resource) context.get(ITransitionConstants.TRANSITION_TARGET_RESOURCE);
 
+      List<Resource> filteredResources = targetResource.getResourceSet().getResources().stream()

```

Comment:
When the target model is named with first characters of the source model and there is some fragment in it, fragments of the sourcemodel are saved and unloaded.

![image](https://user-images.githubusercontent.com/34767327/218482419-0ddf903f-c3e3-4277-b0b2-c0651d87f0a8.png)